CPUMeter.c

```diff
@@ -109,6 +125,19 @@ static void CPUMeter_display(Object* cast, RichString* out) {
          RichString_append(out, CRT_colors[CPU_GUEST], buffer);
       }
    }
+
+   if (this->pl->settings->showCPUTemperature) {
+      char cpuTemperatureBuffer[10];
+      double cpuTemperature = this->values[CPU_METER_TEMPERATURE];
+      if (cpuTemperature < 0)
```

Comment:
Why not prefer `NAN`/`isnan` over magic numbers? Cf. #101.

Reply:
converted